java-symbol-solver-core/src/test/resources/javaparser_expected_output/com_github_javaparser_ast_comments_CommentsParser.txt

```diff
@@ -12,7 +12,7 @@
   Line 51) CommentsCollection comments = new CommentsCollection() ==> com.github.javaparser.ast.comments.CommentsCollection
   Line 51) new CommentsCollection() ==> com.github.javaparser.ast.comments.CommentsCollection
   Line 52) int r ==> int
-  Line 54) Deque prevTwoChars = new LinkedList<Character>(Arrays.asList('z', 'z')) ==> java.util.Deque
+  Line 54) Deque prevTwoChars = new LinkedList<Character>(Arrays.asList('z', 'z')) ==> java.util.Deque<java.lang.Object>
```

Comment:
The more I think about it, the more it feels like having a ReferenceTypeUsage that has no type arguments but has a declaration that has type parameters is implicitly a raw type. We could store it like that (and therefore revert the changes made on this branch). 
As a consequence we'd have to consider this case everytime we test if types are compatible.